1-js/02-first-steps/09-alert-prompt-confirm/article.md

```diff
@@ -1,109 +1,111 @@
+let test = prompt("Teste");
 ```
 
-So, for prompts to look good in IE, we recommend always providing the second argument:
+portanto, para que os prompts tenham boa aparência no IE, recomendamos sempre fornecer o segundo argumento:
 
 ```js run
-let test = prompt("Test", ''); // <-- for IE
+let test = prompt("Teste", ''); // <-- para IE
 ```
 ````
 
 ## confirm
 
-The syntax:
+Sintaxe:
 
 ```js
 result = confirm(question);
 ```
 
-The function `confirm` shows a modal window with a `question` and two buttons: OK and CANCEL.
-
-The result is `true` if OK is pressed and `false` otherwise.
-
-For example:
+A função `confirm` mostra uma janela modal com uma `question` e dois botões: OK e CANCEL.
+O resultado é `true` se OK for pressionado e `false` caso contrário
+Por exemplo:
 
 ```js run
-let isBoss = confirm("Are you the boss?");
+let isBoss = confirm("Você é o chefe ?");
 
-alert( isBoss ); // true if OK is pressed
+alert( isBoss ); // true se OK for pressionado
 ```
 
-## Summary
+## Sumário
+
+Cobrimos 3 funções específicas do navegador para interagir com o visitante:
+
 
-We covered 3 browser-specific functions to interact with visitors:
 
 `alert`
-: shows a message.
+: mostra uma mensagem.
 
 `prompt`
-: shows a message asking the user to input text. It returns the text or, if CANCEL or `key:Esc` is clicked, `null`.
+: mostra uma mensagem pedindo para o usuário inserir texto. Ele retorna o texto ou, se CANCEL ou `key:Esc` for clicado, `null`.
+
+
 
 `confirm`
-: shows a message and waits for the user to press "OK" or "CANCEL". It returns `true` for OK and `false` for CANCEL/`key:Esc`.
+: mostra uma mensagem e espera o usuário pressionar  "OK" ou "CANCEL". Ele retorna `true` para OK e `false` para CANCEL/`key:Esc`.
 
-All these methods are modal: they pause script execution and don't allow the visitor to interact with the rest of the page until the window has been dismissed.
+Todos esse métodos são modais: eles pausam a execução do script e não permitem o visitante interagir com o resto da página até que a janela seja descartada.
 
-There are two limitations shared by all the methods above:
+Existem duas limitações compartilhada entre esses metódos acima:
 
-1. The exact location of the modal window is determined by the browser. Usually, it's in the center.
-2. The exact look of the window also depends on the browser. We can't modify it.
+1. A localização exata da janela modal é determinada pelo navegador. Geralmente, está no centro.
+2. A aparência exata da janela também depende do navegador. Nós não podemos modificá-la.
 
-That is the price for simplicity. There are other ways to show nicer windows and richer interaction with the visitor, but if "bells and whistles" do not matter much, these methods work just fine.
+Esse é o preço da simplicidade. Existem outras duas maneiras de mostrar janelas mais agradáveis e interações mais ricas com os visitantes, mas se "sinos e assobios" não importam muito, esses métodos funcionam bem.
```

Comment:
- "<s>Esse</s> Este é o preço..."
(though correct, would you kindly change to that word?)
- "Existem <s>duas</s> outras maneiras..."